src/doc/trpl/ownership.md

```diff
@@ -121,9 +121,9 @@ let v2 = v;
 ```
 
 The first line creates some data for the vector on the [stack][sh], `v`. The
```

Comment:
> The first line creates some data for the vector on the [stack][sh], `v`.

> The vector’s data, however, is stored on the [heap][sh], and so the vector contains a pointer to that data

The use of the word data between these two lines seems like it could very easily cause confusion. Somebody who is not familiar with Vector's, or stack/heaps could easily misinterpret **Creates some data for the vector** and **vector’s data** as being the same thing here.


I think it would be better as:

> The first line allocates memory for the vector itself and its data. The vector itself is stored on the [stack][sh]. The vector’s data ([1, 2, 3]), however, is stored on the heap and the vector just contains a pointer to that data.

Thoughts?